test/tests/haxe-hello-world/Container.hx

```diff
@@ -0,0 +1,5 @@
+class Container {
+    static function main():Void {
+        trace("Hello World!");
+    }
+}
```

Comment:
GitHub notes that there's "No newline at end of file" -- we'd prefer to have that newline included (happy to make this change myself if you'd rather).

Also, I think this directory needs an `expected-std-out.txt` (similar to `ruby-hello-world`, for example).